### DISCUSSION TITLE
commerical use

### DISCUSSION THREAD

**Discussion by janglichao** [hpcaitech/ColossalAI#3495] [Community | Q&A] [answered]
Hi，I just want to know if I make a commerical project base on colossal AI models，is that allowed？I mean is there any rules I should noticed and comply to？

**Comment by binmakeswell** [comment, marked as answer]
Hi @janglichao Colossal-AI itself follows the [Apache License 2.0](https://github.com/hpcaitech/ColossalAI/blob/main/LICENSE) and has no restrictions on commercial use. You only need to focus on limitations like LLamad that come from other projects.
We also provide professionals for formal collaboration, if you need it. Thanks.
